sdk/core/azure_core/src/http/pager.rs

```diff
@@ -408,8 +445,59 @@ impl<P> PageIterator<P> {
     ) -> Self {
         Self {
             stream: Box::pin(stream),
+            next: Default::default(),
         }
     }
+
+    /// Advance the `PageIterator` to the page referenced by `next_link`.
+    ///
+    /// You should call this before iterating the [`Stream`] or results may be unpredictable.
+    ///
+    /// # Examples
+    ///
+    /// Using a result of a call to [`PageIterator::next_link`] in another process, you can create a new `PageIterator`
+    /// that, when first iterated, will get the next page of results.
+    ///
+    /// ``` no_run
+    /// use azure_identity::DefaultAzureCredential;
+    /// use azure_security_keyvault_secrets::SecretClient;
+    /// use futures::stream::TryStreamExt as _;
+    ///
+    /// # #[tokio::main]
+    /// # async fn main() -> azure_core::Result<()> {
+    /// let client = SecretClient::new("https://my-vault.vault.azure.net", DefaultAzureCredential::new()?, None)?;
+    /// let mut pager = client.list_secret_properties(None)?
+    ///     .into_pages()
+    ///     .with_next_link("next_link_from_another_pager".to_string());
+    ///
+    /// while let Some(secrets) = pager.try_next().await? {
+    ///     let secrets = secrets.into_body().await?;
+    ///     for secret in secrets.value {
+    ///         println!("{:?}", secret.id);
+    ///     }
+    /// }
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn with_next_link(self, next_link: String) -> Self {
```

Comment:
My only minor concern here is that Cosmos calls it a `continuation` (because that's what the header itself is called) and not a `next_link`. Having said that, if Cosmos is the outlier here, I'm OK with that being handled in documentation.

Reply:
I'm starting to like "continuation" more and more myself as well. Seems a nextLink could be a continuation, but a continuation can't be a nextLink.

Reply:
I've changed this across the repo and will update the emitter to use `continuation` now for the `PagerResult::More` variant (just to be consistent).